api/services/containers/v1/containers.proto

```diff
@@ -65,6 +65,9 @@ message Container {
 	// Spec to be used when creating the container. This is runtime specific.
 	google.protobuf.Any spec = 5;
 
+	// Snapshotter specifies the snapshotter name used for rootfs
+	string snapshotter = 6;
```

Comment:
- What would be future additional options? Unlike runtimes, snapshotters are instantiated on the daemon startup.
- If we allow snapshotter options here, do we need to modify snapshots.proto as well?